src/ophys_etl/transforms/classification.py

```diff
@@ -331,11 +331,19 @@ def load_model(classifier_model_uri: str) -> Any:
     return model
 
 
-def main(parser):
-    with open(parser.args["roi_masks_path"], "r") as f:
+def filtered_roi_load(roi_masks_path):
```

Comment:
May want to add a unit test for this function

Reply:
good call.
will wait to add until we have consensus we want this behavior.

Reply:
done